osd/infranode_resized_auto.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "_tags": ["t_config"],
+    "service_name": "SREManualAction",
+    "summary": "Infra nodes resized",
+    "description": "SRE has resized the infra nodes of your cluster to ${INSTANCE_TYPE} to accommodate cluster load.",
```

Comment:
I think we should add a link to these docs https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#node-scaling-after-installation_rosa-limits-scalability as well. Something like "For more info..." or w/e